### PR TITLE
Us filter search

### PR DESCRIPTION
**Pull request by MareikeFla**

### CODE REVIEW

**Review by onemanwenttomow — CHANGES_REQUESTED** [review]
Wow. Lots going on here. Really nice work. I added a few thoughts and suggestions.

File: components/EventDetail/EventDetail.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ export default function EventDetail({ event, mutateEvent }) {
 
   return (
     <>
-      {image.url !== "" ? (
+      {image && image?.url !== "" ? (
```
[the remark this answers is not in the document]
Comment (MareikeFla) [reply components/EventDetail/EventDetail.js:70]: I also thought so, but for some reason doing it like this "{image?.url !== "" ? (" 
will lead to this result:

Screenshot 2024-05-03 at 15 26 25

I would be happy to understand why!

File: components/Search/SearchCard.js [hunk 2]
```diff
@@ -23,6 +48,65 @@ export default function SearchCard({
...
+            </FilterReset>
+          ) : null}
+        </ListRow>
+        <ListRow>
+          {a11yIcons.map((icon, i) => {
+            const A11yIcon = icons[icon.icon];
+            const isSelected = a11yFilter[icon._id];
+            return (
+              <IconWrap
+                key={i}
```
[the remark this answers is not in the document]
Comment (MareikeFla) [reply components/Search/SearchCard.js:72]: fixed it

**Comment by MareikeFla** [comment]
Hi! Thanks for your review! 
Yes it was quite a ride to do this, there where so many edge cases I did not think of in the beginning.

File: components/EventDetail/EventDetail.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (onemanwenttomow) [reply components/EventDetail/EventDetail.js:70]: Hmmm i'm not sure. What is the value of the src of the image in this case?

**Review by onemanwenttomow — APPROVED** [review]
LGTM 🎉

File: components/EventDetail/EventDetail.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (MareikeFla) [reply components/EventDetail/EventDetail.js:70]: image is undefined but in my understanding this "image?.url !== """ should check that?

**Pull request merged**